sync/src/main/java/com/oxeanbits/sync/ui/ForecastMarkerView.kt

```diff
@@ -0,0 +1,39 @@
+package com.oxeanbits.sync.ui
```

Comment:
Is this module used?

Reply:
Yes, the `sync `module is used by the `app `and `example `module. And your `ui` is the point detail view when the chart is pressed.

Reply:
In this case we need to rename its package to:

```suggestion
package com.oxeanbits.forecastchart.sync.ui
```